### DISCUSSION TITLE
Description field for workflows to be shown in UI

### DISCUSSION THREAD

**Discussion by es-fabricemarie** [community/community#12213] [Actions]
I am writing and more workflows to make full use of Actions, and while it's trivial to add comments in-line, I was wondering if it was possible to assign an official description to a workflow. I searched to no avail.

The goal is of course for action descriptions to be visible in the UI and explain to the user what they do before the user decides trigger it.

**Comment by ethomson** [comment]
Hi @es-fabricemarie - this is a really interesting idea.  Can you give me a few examples of the places that you'd want to see this?  (Screenshots would be great!)

**Comment by wolfe** [comment]
@ethomson --- I concur.  It would be nice if a description is supplied such as:
```
name: Matisse Backend Pull Request
description: Test the backend and, upon success, deploy to ci

on:
  pull_request:
```
Then the UI displays "Test the backend and, upon success, deploy to ci" is shown in a small back font beneath (or to the right of) the blue filename
![image](https://github.com/community/community/assets/41852/300fefc8-83c6-4684-b4c7-da0ec805f4a0)

**Reply by es-fabricemarie** [reply]
Yes. Perfect example. Thanks.

**Comment by ShaikhZayan** [comment]
res 2

**Comment by nullbio** [comment]
Is there any interest in doing this? It'd be nice to display some documentation text under the workflow headings that describe the workflow in more detail.
